src/instrumentation.node.ts

```diff
@@ -0,0 +1,42 @@
+import { OTLPLogExporter } from "@opentelemetry/exporter-logs-otlp-http";
+import { OTLPMetricExporter } from "@opentelemetry/exporter-metrics-otlp-http";
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-http";
+import { HttpInstrumentation } from "@opentelemetry/instrumentation-http";
+import { resourceFromAttributes } from "@opentelemetry/resources";
+import { BatchLogRecordProcessor } from "@opentelemetry/sdk-logs";
+import { PeriodicExportingMetricReader } from "@opentelemetry/sdk-metrics";
+import { NodeSDK } from "@opentelemetry/sdk-node";
+import { SimpleSpanProcessor } from "@opentelemetry/sdk-trace-base";
+import { ATTR_SERVICE_NAME } from "@opentelemetry/semantic-conventions";
+
+const TRACE_URL =
+  process.env.OTEL_EXPORTER_OTLP_TRACES_ENDPOINT ??
+  "http://localhost:4318/v1/traces";
+
+const METRICS_URL =
+  process.env.OTEL_EXPORTER_OTLP_METRICS_ENDPOINT ??
+  "http://localhost:4318/v1/metrics";
+
+const traceExporter = new OTLPTraceExporter({ url: TRACE_URL });
+const metricsExporter = new OTLPMetricExporter({
+  url: METRICS_URL,
+});
+const logsExporter = new OTLPLogExporter({ url: TRACE_URL });
```

Comment:
Does otel intend to deliver logs to the traces endpoint? this is missing some context for me, maybe a comment could clarify